extensions/interactions/EndExploration/EndExploration.html

```diff
@@ -7,7 +7,21 @@
     <div style="color: red;" ng-repeat="expId in invalidExpIds">
       <strong>Error:</strong> no published exploration found with ID '<[expId]>'
     </div>
-    <oppia-end-exploration-recommended-explorations recommended-exploration-ids="recommendedExplorationIds" recommended-exploration-summaries="recommendedExplorationSummaries" collection-id="collectionId"></oppia-end-exploration-recommended-explorations>
+
+    <div layout="row" layout-wrap>
+      <div ng-repeat="expId in recommendedExplorationIds track by $index">
+        <exploration-summary-tile ng-if="recommendedExplorationSummaries[$index]"
+                                  collection-id="collectionId"
+                                  exploration-id="expId"
+                                  exploration-title="recommendedExplorationSummaries[$index].title"
+                                  last-updated-msec="recommendedExplorationSummaries[$index].last_updated"
+                                  objective="recommendedExplorationSummaries[$index].objective"
+                                  thumbnail-image-url="recommendedExplorationSummaries[$index].thumbnail_image_url"
```

Comment:
Yep, definitely. I just pushed another commit that should fully fix this -- PTAL.